lib/lighthouse/veteran_verification/configuration.rb

```diff
@@ -8,8 +8,7 @@ module VeteranVerification
   class Configuration < Common::Client::Configuration::REST
     self.read_timeout = Settings.lighthouse.veteran_verification.timeout || 20
 
-    API_SCOPES = %w[disability_rating.read enrolled_benefits.read flashes.read launch service_history.read
-                    veteran_status.read].freeze
+    API_SCOPES = %w[disability_rating.read].freeze
```

Comment:
Based on my understanding, we currently only have access to the `disability_rating.read` and `launch` scopes. If we include the `launch` scope here, we have to provide the launch context as part of the access token request, so it's best to remove it. NOTE: If we use the `launch` scope and provide both the launch context and the ICN in the request URL, the launch context's contents are not validated